clientd3d/music.c

```diff
@@ -492,51 +654,53 @@ void PlayMusicRsc(ID rsc)
    debug(("PlayMusicRsc %d\n", rsc));
 
    /* If we're already playing same music file, keep jammin' */
-   if (playing_music)
+   if (playing_music && !isMusicPaused)
    { 
-      if (rsc != 0 && bg_music != 0 && 
-          !stricmp(LookupNameRsc(rsc), LookupNameRsc(bg_music)))
+      if (rsc != 0 && bg_music != 0) 
 	  {
-		  debug(("Already playing that music.\n" ));
+        if (!stricmp(LookupNameRsc(rsc), LookupNameRsc(bg_music)))
+        {
+		  debug(("DEBUG Already playing that music.\n" ));
         return;
+        }
+        /* Playing music is true, not paused, need new bg music */
+        /* so kill the current background music before continuing. */
+        #ifdef M59_MSS
+			   AIL_end_sample( hseqBackground );
+        #else
+			   mciSendCommand(midi_bg_music_element, MCI_CLOSE, 0, 0); 
+         #endif
```

Comment:
The indentation looks weird in this section.  It's not worth the trouble to fix everywhere, but maybe you can set your editor to use spaces instead of tabs so you can line things up here?

Reply:
Sorry - I didn't realize how annoying this looks on Github since it looks correct on my machine.

The good news is I can use the Github code view to see where I've messed this up and I will fix it manually.

Reply:
Fixed.